coreos-4/policies/SA-System_and_Services_Acquisition/component.yaml

```diff
@@ -357,50 +357,94 @@
   narrative:
     - key: a
       text: |
-        A control response to SA-5(a) is planned. Progress
-        can be tracked at:
+        Obtaining admin docs are an organization policy and procedure control,
+        and are not applicable to OpenShift.
+        To assist with compliance efforts on behalf of organizations, documentation is
+        referenced below.
+        Documentation progress can be tracked at:
 
         https://issues.redhat.com/browse/CMP-367
+
     - key: a.1
       text: |
-        A control response to SA-5(a)(1) is planned. Progress
-        can be tracked at:
+        CoreOS installation documents:
+        https://coreos.com/docs/#tectonicInstallationHeader
+        CoreOS Hardening guide:
+        https://coreos.com/os/docs/latest/hardening-guide.html
+        Machine Config Operator documentation:
+        https://github.com/openshift/machine-config-operator/tree/master/docs
+
+        Documentation progress can be tracked at:
 
         https://issues.redhat.com/browse/CMP-367
+
     - key: a.2
       text: |
-        A control response to SA-5(a)(2) is planned. Progress
-        can be tracked at:
+        Documention of effective use and maintenance of security functions is
+        an organization policy and procedure control.
+        CoreOS uses SELinux to apply security contexts to files, devices, processes,
+        and containers.
+        The CoreOS hardening guide touches on the basics of effective security tool
+        use and maintenance:
+        https://github.com/coreos/docs/blob/master/os/hardening-guide.md
```

Comment:
url should be https://docs.openshift.com/container-platform/latest/security/container_security/security-hardening.html